arviz/tests/test_plots.py

```diff
@@ -313,6 +313,14 @@ def test_plot_kde_quantiles(continuous_model, kwargs):
     assert axes
 
 
+def test_plot_kde_inference_data():
+    eight = load_arviz_data("centered_eight")

```

Comment:
Docstring for this test would be great even if its oneline. I know we don't have docstrings for all tests but I'm trying to reverse that trend.